auditbeat/module/auditd/audit_linux_test.go

```diff
@@ -349,7 +349,7 @@ func TestBuildMetricbeatEvent(t *testing.T) {
 }
 
 func buildSampleEvent(t testing.TB, lines []string, filename string) {
-	var msgs []*auparse.AuditMessage
+	var msgs []*auparse.AuditMessage //nolint:prealloc // False positive.
```

Comment:
```suggestion
	var msgs []*auparse.AuditMessage //nolint:prealloc
```

Not a FP. Suppressing the prealloc lint is fine as this is a `*_test.go` file.